snafu/trex_wrapper/ci_test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Why even have this when we are not testing in CI?
@mukrishn

Reply:
I will remove this file, i  was not sure at that time.